app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, src_file, dst_file = command.split(" ")
```

Comment:
Consider adding error handling for the `split` operation. If the command string is not in the expected format (e.g., less than three words), it will raise a `ValueError`. You could catch this exception and handle it gracefully.